grails-app/conf/BootStrap.groovy

```diff
@@ -194,13 +194,13 @@ class BootStrap {
                 name: orderSummary?.order?.name,
                 origin: orderSummary?.order?.origin?.name + (origOrgCode ? " (${origOrgCode})" : ""),
                 destination: orderSummary?.order?.destination?.name + (destOrgCode ? " (${destOrgCode})" : ""),
-                orderedOn: orderSummary?.order?.dateOrdered,
+                dateOrdered: orderSummary?.order?.dateOrdered,
                 orderedBy: orderSummary?.order?.orderedBy?.getName(),
                 orderItemsCount: orderSummary?.order?.activeOrderItems?.size()?:0,
                 orderedOrderItemsCount: orderSummary?.order?.orderedOrderItems?.size()?:0,
                 shippedItemsCount: orderSummary?.order?.shippedOrderItems?.size()?:0,
                 receivedItemsCount: orderSummary?.order?.receivedOrderItems?.size()?:0,
-                total: "${orderSummary?.order?.total?.setScale(2, RoundingMode.HALF_UP)} ${orderSummary?.order?.currencyCode ?: defaultCurrencyCode}",
+                total: "${orderSummary?.order?.total ? orderSummary?.order?.total?.setScale(2, RoundingMode.HALF_UP) : 0} ${orderSummary?.order?.currencyCode ?: defaultCurrencyCode}",
```

Comment:
fixed bug that was not "closing" the JSON in the response. The problem was with orders with `total=0`, because `setScale` couldn't work here and the `total?` is only checking if it's not `null`, but 0 is not `null`, so it was proceeding.

Reply:
This is no longer required (I added the `else 0.0` in total)